src/commands/monthly.ts

```
@@ -21,8 +21,8 @@ export const monthlyCommand = define({
 	...sharedCommandConfig,
 	async run(ctx) {
 		// Load configuration and merge with CLI arguments
-		const config = loadConfig(ctx.values.config);
-		const mergedOptions = mergeConfigWithArgs(ctx, config);
+		const config = loadConfig(ctx.values.config, Boolean(ctx.values.debug));
```

ESLint

[error] Unsafe argument of type `any` assigned to a parameter of type `string | undefined`.

(ts/no-unsafe-argument)